### PR TITLE
Crazy4pi314/update readme

### PR DESCRIPTION
**Pull request by crazy4pi314**
TESTING ACTIONS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,35 +3,58 @@
 [![Unitary Fund](https://img.shields.io/badge/Supported%20By-UNITARY%20FUND-brightgreen.svg?style=flat)](http://unitary.fund)
 
 This library implements a variety of different proposals for memory for quantum computers, also commonly called qRAM.
-Want to learn more about what qRAM is?
-Check out the [primer on memory for quantum computers](https://github.com/qsharp-community/qram/blob/docs/docs/primer.pdf) in our docs!
+
+> Want to learn more about what qRAM is?
+> Check out the [primer on memory for quantum computers](https://github.com/qsharp-community/qram/blob/docs/docs/primer.pdf) in our docs!
```
Comment (RolfHuisman) [inline README.md]: Need to check if this url still works afterwards.

File: README.md [hunk 2]
```diff
@@ -3,35 +3,58 @@
...
+- **Can we design an efficient qRAM?**
+
+  _Maybe_.
+ In this primer we'll take a look at proposals that will in principle run in polynomial depth, and others that scale far worse.
+ There are some very interesting qubit-time tradeoffs one can explore, in particular if the data being stored has some sort of underlying structure.
+ Regardless, even if we can design an efficient circuit, we'd also like something that is efficient in a fault-tolerant setting, and this is potentially very expensive.
+- **Can I build one?**
+
+  _Maybe_.
+  No one has actually done so, but there are a handful of hardware proposals that will be discussed in more detail in the hardware section of [the primer on memory for quantum computers](https://github.com/qsharp-community/qram/blob/docs/docs/primer.pdf).
```
Comment (RolfHuisman) [inline README.md]: Url check 2.

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -38,7 +43,7 @@ jobs:
         # The message for the commit
         message: "Updating primer.pdf"
         # Name of the branch to use, if different from the one that triggered the workflow
-        ref: docs
+        #ref: docs
```
Comment (RolfHuisman) [inline .github/workflows/docs.yml:45]: We may need to bump the version to 4.4.0 for this to work.

Comment (crazy4pi314) [reply .github/workflows/docs.yml:45]: Bumping now to check

**Pull request merged**